lib/stimulus_reflex/reflex.rb

```diff
@@ -60,7 +61,8 @@ def initialize(channel, url: nil, element: nil, selectors: [], method_name: nil,
     @params = params
     @permanent_attribute_name = permanent_attribute_name
     @broadcaster = StimulusReflex::PageBroadcaster.new(self)
-    @reflex_id = reflex_id
+    @client_attributes = OpenStruct.new(client_attributes)
```

Comment:
I'm splitting hairs now here, but could we make that an _actual_ struct so there's a clear definition of what can go in there?